CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+
+## Sending a pull request
+Please submit pull requests using a forked repo and open pull requests against the `edge` branch (the default) unless otherwise instructed.
+
+When opening a pull request, the form will be pre-populated with our template. Please fill out the template to provide structure to your PR message. If you've already written a good commit message (see below) it will be easy to use with our template.
+
+A pull request will need to pass the following checkpoints to be accepted:
+
+- Initial review: a maintainer will review your summary and make sure an appropriate issue is linked
+- Testing: automated tests will run against your changes
+- Code review: you will get feedback from a maintainer or other contributors in the form of comments
+
+## Writing a good commit message
+
+We value good commit messages that are descriptive and meaningful at a glance. A good format to follow is like the following:
+
+```txt
+<short description>
+
+Fixes: #<issue>
+
+<a longer description that includes>
+
+- a summary of the changes being made
+- the rationale for the change
+- (optional) anything tricky or difficult as a heads up for reviewers
+- (optional) additional follow up work that should be done (with links)
+```
+
+We **squash** pull-requests as part of the merge process, which means that intermediate commits will have their messages appended. We prefer to have a single commit in the git history for each PR.
+
+## Code review
+
+The maintainers or other contributors will add comments to your pull request giving feedback, asking questions, and making suggestions. Please respond to these comments to either continue the discussion or explain whether or not you plan to address the feedback. Ultimately, accepting a pull request is at the maintainer's discretion.
+
+### Being proactive 
+
+It can be helpful for you to comment on your own PR to point out relevant locations, decisions, opportunities for feedback, and tricky parts. This will help reviewers focus their attention as well as save them time.
+
+### Resolving Feedback
+
+You can "resolve" comments on your pull request when you've addressed the feedback: either through discussion or through making a code change. As the contributor of the pull-request feel free to mark comments as resolved when you feel like you've done a reasonable job addressing the feedback.
+
+If you are the code reviewer, it's your responsibility to follow up (politely) if you feel your feedback has not been addressed adequately.
+
+### Participating in code review
+
+We welcome **any contributor or community member** to engage with **any pull request** on our repository. Feel free to make suggestions for improvements and ask questions that are relevant. If you're asking questions for your learning, please make it clear that your questions are "non-blocking" for the pull request.
+
+See the [code reviewing documentation](../contributing-code/contributing-code-reviewing/README.md) for guidance on code reviewing.
```

Comment:
This link will be broken, let's update it to an fully qualified URL to the radius repo

Reply:
Good catch

Reply:
Updated. Thanks for reviewing! Let me know if anything else looks off.